app/javascript/lib/scp-api.js

```diff
@@ -324,13 +324,18 @@ export async function deleteStudyFile(studyAccession, fileId, mock=false) {
  * @param {String} bucketName bucket name
  * @param {String} fileName file name
 */
-export async function fetchBucketFile(bucketName, fileName, mock=false) {
+export async function fetchBucketFile(bucketName, fileName, maxBytes=null, mock=false) {
```

Comment:
Plausible!  Streaming parallel validation would likely reduce sync CSFV times by 2-5x.

That might be a good case for doubling `MAX_SYNC_CSFV_BYTES` as you suggest, but we'd still want some maximum download size.  The percentiles for files by size drop off sharply beyond 50 MB, yet a significant portion of files are much bigger.   E.g. [> 20% of dense matrices are > 1 GB](https://mixpanel.com/s/4klxkX).

Streaming parallel validation eliminates file size as a concern for upload CSFV, but not sync CSFV.  That's because bytes get sent up from the client _regardless of CSFV_ in the upload UI, but bytes get sent down to the client _only for CSFV_ in the sync UI.